README.md

```diff
@@ -43,6 +43,7 @@ The React render happens on both the server and the client using shared code. Re
 
 ```
 npm install
+npm run build
```

Comment:
After I cloned the project and npm install & start the browser reported a 404 for the referenced static file.